src/DeviceRenderer.js

```diff
@@ -298,8 +299,8 @@ module.exports = class DeviceRenderer {
     disconnect() {
         this.initialized = false;
 
-        if (typeof this.camera !== 'undefined' && typeof this.camera.getClientVideoStream !== 'undefined') {
-            this.removeLocalStream(this.camera.getClientVideoStream());
+        if (typeof this.mediaManager !== 'undefined') {
```

Comment:
`typeof this.mediaManager !== 'undefined' && this.mediaManager.disconnect();`

Is more sexy, isn't ?

Reply:
more sexy but less readable so I'm not sure if it's better in this case :/